leasing/models/rent.py

```diff
@@ -753,6 +830,14 @@ def is_active_on_period(self, date_range_start, date_range_end):
 
         return False
 
+    def set_start_price_index_point_figure(self):
+        if self.old_dwellings_in_housing_companies_price_index:
+            start_index_number_yearly = IndexPointFigureYearly.objects.get(
+                index=self.old_dwellings_in_housing_companies_price_index,
+                year=self.start_date.year - 1,
+            )
+            self.start_price_index_point_figure = start_index_number_yearly.point_figure
```

Comment:
Thanks for the naming changes! Should this line `start_index_number_yearly.point_figure` be `start_index_point_figure_yearly.value`, because we changed the column name to `value`?

Reply:
Fixed in a later commit